.gitignore

```diff
@@ -186,3 +186,5 @@ testem.log
 .DS_Store
 Thumbs.db
 *.vscode*
+gemini/sample-apps/finance-advisor-spanner/gcs-bucket.zip
```

Comment:
Can you remove this? You should only use the files direct from GCS

Reply:
please provide me the bucket link

Reply:
done